lib/vsc/install/shared_setup.py

```diff
@@ -361,13 +373,15 @@ def get_name_url(self, filename=None, version=None, license_name=None):
         if len(res) != 3:
             raise Exception("Cannot determine name, url and download url from filename %s: got %s" % (filename, res))
         else:
+            keepers = dict()
```

Comment:
`dict()` -> `{}` (again for consistency with existing code, see `res = {}` above)